src/hostname/hostnamed.c

```diff
@@ -212,7 +212,7 @@ static const char* fallback_chassis(void) {
            unreliable enough, so let's not do any additional guesswork
            on top of that.
 
-           See the SMBIOS Specification 4.0 section 7.4.1 for
+           See the SMBIOS Specification 3.0 section 7.4.1 for
```

Comment:
I'm having a hard time understanding exactly what you are referring to, according to https://www.dmtf.org/standards/smbios the latest publicized release is 3.0.0 which is linked below. Should I change it to 3.0.0?